README.md

```diff
@@ -0,0 +1,31 @@
+# Reyan
+ Reyan is a free modern open-source application based on the latest feature of the android/IOS development world which'll let you recite the holy quran easier than ever!
+
+## Tools And Technolagies
+* programming language: Kotlin
+* Architecture: MVI
+* Kotlin multi-platform
+* MultiThreading: Coroutines
+* Android DI: Dagger2
+* General networking: ktor.http
+* Android networking: OkHttp engine
+* Misc: ExoPlayer
+
+## About
+  Reyan means "**Gift of God**" and Reyan applicaton, represents the most valuable gift of God which is The Holy Quran.
+  
+  Our goal is to make, reading the holy quran and other religious books such as nahj al-Balagha, Mafatih al-janan, etc. easier than ever.

```

Comment:
let's keep it to Quran exclusively.
(not sure when we can add other books right now)